handbook/ops/roles/business_operations_analyst.md

```diff
@@ -0,0 +1,50 @@
+# Business Operations Analyst
+
+Sourcegraph’s business operations team empowers every part of the business with the data and insights, execution and project management, and strategic and business expertise that they need to make better decisions and move faster. As an early hire of the business operations team, you will be a vital member of both the business operations team and the company as we grow.
+
+Our team and our sales are growing fast—we’ve proven strong product-market fit, with wall-to-wall usage and top-tier engagement, genuine love from our users and customers, and tens of thousands of paid daily active users at some of the world’s premier development teams.
+
+## Responsibilities
+
+Your day-to-day responsibilities will include:
+* Partner with teams across the company to identify key metrics and growth drivers, build and maintain data pipelines and dashboards, and recommend actions to drive growth.
+* Perform analyses and research that contribute to important strategic and operational decisions
+* Design and implement the business systems and processes that will scale with as the company does.
+* Pitch in however you can when needed to drive the company forward. We're still a lean team and are often asked to fill gaps in the growing organization
+
+Your first project could be:
+* Leading a deep-dive into retention users on [Sourcegraph Cloud](https://sourcegraph.com/search). Why do users come back or churn? What changes in our product would you recommend to optimize for retention? 
+* Supporting the search team in evaluating our search performance, including finding baselines, peeling back layers to understand which dimensions of searches lead poor performance.
+* Developing dashboards to understand and track the adoption of [Sourcegraph extensions](https://sourcegraph.com/extensions), and working with the web and product teams on how to make extensions more discoverable. 
+
+## Qualifications
```

Comment:
Anything to put here around familiarity with tools/languages/analytical frameworks? e.g. SQL? Looker? R? Etc. Might help people determine if they're a good fit. We can caveat all of them with "(nice to have)" or put it into a separate section, if helpful.